src/components/o-datepicker/js/datepicker.js

```diff
@@ -126,6 +126,9 @@ export default class Datepicker {
   }
 
   isItemInLowerHalf(target) {
+    if (!target) {
```

Comment:
in the constructor there's a little mistake. change `this._localValue` to `this.localeValue`.